api-service/src/controllers/CreateQueryTemplate/QueryTemplateValidator.ts

```diff
@@ -0,0 +1,57 @@
+import * as _ from "lodash";
+import { config } from "../../configs/Config";
+
+export const validateTemplate = async (req: Request) => {
```

Comment:
What is the design for this file ? are we checking that query is having the variables only or something else ?

Reply:
yes